templates/details/_details-header.html

```diff
@@ -80,21 +80,36 @@ <h1 class="p-media-object__title">
       </div>
     </div>
     <div class="col-4">
-      <p style="margin-bottom: 0.5rem;"><small class="u-no-padding--top u-text--muted">Platform:</small></p>
-      {% if package.store_front["deployable-on"]|length == 1 and package.store_front["deployable-on"][0] == "kubernetes" %}
-      <img src="https://assets.ubuntu.com/v1/9f8a8273-2018-logo-kubernetes+%282%29.svg" alt="" width="160" height="27" class="p-image--base">
-      {% else %}
-      <img src="https://assets.ubuntu.com/v1/adac6928-ubuntu.svg" alt="" width="89" height="20" class="p-image--base">
-      <div class="series-tags">
-        {% for version in package.store_front.series %}
-        {% if version != "Kubernetes" %}
-        <span class="series-tag">
-          {{ version }}
-        </span>
-        {% endif %}
-        {% endfor %}
-      </div>
-      {% endif %}
+      <p style="margin-bottom: 0.5rem;">
+        <small class="u-no-padding--top u-text--muted">
+          Platform:
+        </small>
+      </p>
+
+      {% for base in channel_bases %}
```

Comment:
We shouldn't add this new variable `channel_bases` into the template, this will work on the overview tab but not other tabs:
https://charmhub-io-1178.demos.haus/slurmd/docs

My next comment is also related to this.